sys/vfs/vfs.c

```diff
@@ -867,6 +868,13 @@ static inline int _allocate_fd(int fd)
 {
     if (fd < 0) {
         for (fd = 0; fd < VFS_MAX_OPEN_FILES; ++fd) {
+            if ((fd == STDIN_FILENO) || (fd == STDOUT_FILENO) || (fd == STDERR_FILENO)) {
```

Comment:
POSIX defines them as 0, 1 and 2. (e.g. here: http://pubs.opengroup.org/onlinepubs/000095399/functions/stdin.html). Can't we just start counting with 3?

Reply:
(sorry, didn't realise the two ACKs.) ACK.